app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="container">
+  <h2 class="text-center pt-3 pb-2">Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="form-group">
+      <%= f.text_field :full_name, placeholder: 'Your name here', class: 'form-control', autofocus: true %>
+    </div>
+
+    <div class="form-group">
+      <%= f.text_field :username, placeholder: 'Your username here', class: 'form-control' %>
+    </div>
+
+    <div class="form-group">
+      <%= f.email_field :email, placeholder: 'Your email here', class: 'form-control' %>
+    </div>
+
+    <div class="form-group">
+      <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %>
```

Comment:
- [ ] Create a helper method and render the markup within the helper method instead of doing it straight in the view